docs/src/app/components/pages/components/Slider/Page.js

```diff
@@ -23,6 +25,7 @@ const descriptions = {
   'increments.',
   value: 'The slider bar can have a set minimum and maximum, and the value can be ' +
   'obtained through the value parameter fired on an onChange event.',
+  axis: 'The orientation of the slider can be reversed and rotated using the `axis` prop.',
```

Comment:
We are migrating the description closer to the examples. What do you think about following [this example](https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/Avatar/ExampleSimple.js#L20)?